doc/adr/2024-08-13-googleapis-sha-update-policy.md

```diff
@@ -0,0 +1,20 @@
+**Title**: Releases should contain googleapis SHA that are recent and mature.
```

Comment:
comment: "recent" and "mature" are opposites. But I get that the point is we are striking a balance between the two.